annotations.md

```diff
@@ -26,4 +26,6 @@ This specification defines the following annotation keys, intended for but not l
 * **org.opencontainers.image.revision** Source control revision identifier for packaged software.
 * **org.opencontainers.image.vendor** Name of the distributing entity, organization or individual.
 * **org.opencontainers.image.licenses** Comma-separated list of licenses under which contained software is distributed, in [SPDX Short identifier]https://spdx.org/licenses/] form.
-* **org.opencontainers.image.ref.name** Name of the reference for a target (string). SHOULD only be considered valid when on descriptors on `index.json` within [image layout](image-layout.md).
+* **org.opencontainers.image.ref.name** Name of the reference for a target (string).
+  * SHOULD only be considered valid when on descriptors on `index.json` within [image layout](image-layout.md).
+  * Character set of the value SHOULD conform to the [Uniform Resource Identifier (URI) RFC3986](https://tools.ietf.org/html/rfc3986).
```

Comment:
As this stands, it sounds like you're punting to the `URI` rule, but that requires a `scheme`, colon, and `hier-part`.  I doubt we want to recommend all names contain a colon.  Can we punt to `hier-part` instead?  Or maybe `segment-nz`?  @stevvooe mentioned “[schema-less URIs][1]”, perhaps he was thinking of `path-noscheme`?

[1]: https://github.com/opencontainers/image-spec/pull/591#issuecomment-285153069